oresat_star_tracker/client/star_tracker_client.py

```diff
@@ -0,0 +1,232 @@
+import traceback
```

Comment:
This file looks like it is based off the OLAF read/write script. That script was design to not use the EDS files. Since the star tracker has an eds file most of these can be replace with `node.sdo[INDEX][SUBINDEX].phys`